### PR TITLE
Add desktop viewer, tui, tails and aspire dashboard to registry

### PR DESCRIPTION
**Pull request by svrnm**
This PR adds some commonly used OTel viewers to the registry:

* otel-desktop-viewer by @CtrlSpice
* otel-tui by @ ymtdzzz
* tails by @jaronoff97
* Aspire Dashboard by Microsoft (cc @samsp-msft)

### CODE REVIEW

**Review by mx-psi — APPROVED** [review]

File: data/registry/tools-elixir-tails.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - webserver
+  - tailing
+urls:
+  repo: https://github.com/jaronoff97/tails
+license: Apache 2.0
+description:
+  Tails is a small webserver written in Elixir that listens on a socket and
+  streams live messages from a collector.
+authors:
+  - name: jaronoff97
```
Comment (jaronoff97) [inline data/registry/tools-elixir-tails.yml:15]: github this is my handle, you know who i am :sob:

Comment (svrnm) [reply data/registry/tools-elixir-tails.yml:15]:
```suggestion
  - name: Jacob Aronoff
```

Comment (svrnm) [reply data/registry/tools-elixir-tails.yml]: fixed

**Review by jaronoff97 — APPROVED** [review]
thank you @svrnm :bow:

**Review by codeboten — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
well that was weird, if the refcache gets hoarky I'll fix it up some more I guess

**Pull request merged**